### ISSUE TITLE
Any progress with the get_bid_price() function? 

### ISSUE THREAD

**Issue by delica1** [a-wakeel/PyDEGIRO#3]
Thanks for this API, very useful! Could you get the price ticker info working? I tried looking at the stream in Chrome but it doesn't even show up in the network tab.

Almir

**Comment by a-wakeel** [comment]
Talking about **get_ask_bid_price()** function?
I can work on it if managed to find a free time. You can contribute if you want.

Let me know if you need any help regarding the contribution.

**Comment by a-wakeel** [comment]
I guess i was facing some problem in reverse engineering the api, if you could reverse it that would make it pretty simple to develop.

**Comment by delica1** [comment]
I would like to, but not sure where to start. Is this a websocket
connection? How are you viewing the network traffic? I don't see anything
in chrome developer tools.

**Comment by a-wakeel** [comment]
You will not see anything in chrome or any browser developer tools because this is pure python based, so you will see the responses in python console instead, if not already printed to stdout you can use simple **print** statement.

Its based on **python requests** module, you can see it in the source file.

For creating a new functions you have to understand the flow of the lib for example:
first you need to create a class instance by:
```python
degiro = DegrioApi('username', 'password')
```

then you need to login by:
```python
session = degiro.login()
```

the login function will return you a session object which then you can use it in your own function  to send **GET** or **POST** request.

As an example one of the function from the api:
```python
 def get_portfolio(self, session):
        update_url = '{trading_url}v5/update/'.format(trading_url=self.trading_url)
        res = session.get(
            "{update_url}{account_id};""jsessionid={jsession_id}".format(
                update_url=update_url,
                account_id=self.account_no,
                jsession_id=self.session_id),
            params={'portfolio': 0, 'totalPortfolio': 0})
        if res.status_code == requests.codes.ok or res.status_code == 201:
            return res.json()
        else:
            return '{"Error":"Could not fetch portfolio", "StatusCode":{status_code}}'.format(
status_code=res.status_code
```

**Comment by delica1** [comment]
Thanks for the info! I understand the python code, but I cannot see how the
Degiro website is getting the latest price info, since I do not see any
network traffic. I will try the firefox debugger, maybe this will show how
the Degiro website is updating the price ticker. Unless you have a better
suggestion!

**Comment by a-wakeel** [comment]
Yeah that is the thing to do. You will notice alot of requests on browser tools, you have to find that exact one which you need, once you found it the rest is straight.
But it will be better and make your way easy if you simulate the same behavior as browser in some api testing tool like **postman**

**Comment by a-wakeel** [comment]
We will remain this issue open until its fixed.
